SAMtags.tex

```diff
@@ -411,17 +411,111 @@ \section{Locally-defined tags}
 
 \begin{appendices}
 \appendix
-\section{SAM Tags History}\label{sec:history}
+\section{Tag History}
 
-This lists the date of each tagged SAM version along with changes that
-have been made while that version was current.  
+This appendix lists when standard tags were initially defined or significantly changed, and other historical events that affect how tags are interpreted or what files they may appear in.
 
-\subsection*{1.5: 23 May 2013 to current}
-\begin{itemize}
-\item Cellular barcode tags CB, CR, and CY added. (May 2018)
-\item Add UMI-related tags (RX, QX, OX, BZ, MI) and clarified usage of sample barcode tag BC. (August 2017)
-\item SAMtags.txt (this file) created with tags from SAMv1 
-\end{itemize}
+\setlength{\parindent}{0pt}
+\newcommand*{\gap}{\vspace*{2ex}}
+
+\subsubsection*{May 2018}
+
+Cellular barcode tags CB, CR, and CY added.
+
+\subsubsection*{November 2017}
+
+CG tag added.
+
+\subsubsection*{August 2017}
+
+Unique molecular identifier tags BZ, MI, OX, QX, and RX added.
+
+Usage of sample barcode tag BC clarified.
+
+\subsubsection*{June 2017}
+
+Corrected the description of the E2 tag, which previously wrongly claimed E2 to be encoded as per {\sf QUAL}.
```

Comment:
why do we want to record the incorrect claim here?

Reply:
The first half of the sentence records a substantive change to the spec, and the second half answers the question that everyone reading the first half will have: _What was it that changed?_

I was going to reply that the previous text was not self-consistent so obviously wrong to interested readers… therefore we should just delete this entry.

However (see 434cda9238e9099418362016380460d35b0dc380) as the `\tagvalue` was also just plain wrong, I think this was sufficiently consistently wrong that it's worth listing the correction. Will change it to _which was previously unclear whether E2 contains bases or base qualities_.